src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -838,11 +838,22 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
 
     private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container)
     {
-        $container
+        $definition = $container
             ->getDefinition('property_accessor')
             ->replaceArgument(0, $config['magic_call'])
             ->replaceArgument(1, $config['throw_exception_on_invalid_index'])
         ;
+
+        if (isset($config['cache']) && $config['cache']) {
+            $container->setParameter(
+                'property_accessor.cache.prefix',
```

Comment:
How does this work with a cache warmer? The cache warming is done in a different kernel.root_dir AFAIK. So if I write a cache warmer that initializes the property access cache for the property paths that I use, it will be useles or? As soon as the project is then moved to the real directory, all caches are invalidated as the namespace changed?